src/components/DeclareForm/DeclareForm.js

```diff
@@ -29,6 +28,7 @@ class DeclareForm extends Component {
   }
 
   handleAddImage = (images) => {
+    console.log('length: ', images.length);
```

Comment:
All console.log should be eliminated as it's not a good practice and make the code ugly

Reply:
Sorry I forgot to delete this